filters/exclusions.txt

```diff
@@ -257,5 +257,10 @@ _campaign=$popup
 ||github.io^$third-party,xmlhttprequest,domain=~facebook.com|~gametactic.org|~github.com|~github.io|~tumblr.com|~wottactic.com
 !###########################
 !
+!### NoCoin Filter List ###
+! https://github.com/AdguardTeam/AdguardFilters/issues/84173
+##.ad
```

Comment:
better regexp, like `/##\.an-advert-banner$/`

Reply:
without `^`

Reply:
How about `/^##\.ad$/`?

Reply:
It seems `^` is not needed.